### PR TITLE
chore: use gha instead of travis

### PR DESCRIPTION
**Pull request by lazysoundsystem**
Refs: OPS-9006

for linting and phpcs - no actual tests as there are no custom modules

travis.yml should be removed after this is merged

### CODE REVIEW

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+          php-version: 8.1
+          tools: composer
+        env:
+          fail-fast: true
+
+      - name: Setup PHP with PECL extension locally
+        uses: shivammathur/setup-php@v2
+        if: ${{ env.ACT }}
+        with:
+          php-version: 8.0
```
Comment (cafuego) [inline .github/workflows/run-tests.yml:23]: The tests should run on the same PHP version as the docker image used by the site; that way we can catch syntax issues and composer depend issues.

Comment (lazysoundsystem) [reply .github/workflows/run-tests.yml:23]: I was wondering about this - the site is using 8.0 - https://github.com/UN-OCHA/common-design-site/blob/0332cb7a64ea3c27c1593125a99552b4c3d459a7/docker/Dockerfile#L7 - but is there a way to use an environment variable so we don't have to change it here when we change it in the dockerfile?

Comment (cafuego) [reply .github/workflows/run-tests.yml:23]: We might be able to set an env var (using an action) based on the Dockerfile.

**Comment by rupl** [comment]
Whoops I got excited and deleted Travis before seeing there was unresolved conversation. I'll hold off on merging this for today's deploy.

**Comment by cafuego** [comment]
The unresolved conversation can perhaps be solved via https://github.com/UN-OCHA/actions/pull/24 so you *could* delete Travis and we can sort the versioning in a follow-up PR :-)

**Comment by rupl** [comment]
Travis is gone so I will merge it and yolo our way back to CI coverage.

**Pull request merged**